flow-typed/types/smart-switch-config.js

```diff
@@ -0,0 +1,51 @@
+// @flow
+/**
+ * @typedef SmartSwitchConfig
+ * @property {string} accountCode - Your YOUBORA account code.
+ * @property {string} application - The application code configured in the UI.
+ * @property {number} responseTimeoutSec - response timeout in seconds.
+ * @property {QueryParams} optionalParams - optional query params.
+ */
+declare type SmartSwitchConfig = {
+  accountCode: string,
+  application: string,
+  responseTimeoutSec?: number,
+  optionalParams?: queryParams
+};
+
+/**
+ * @typedef QueryParams
```

Comment:
as this config is passthrough to the sdk, you can just refer to the sdk api for optionalParams, instead of documenting them here. so we dont need maintain it